cfgov/unprocessed/css/pages/careers.less

```diff
@@ -21,8 +21,38 @@
     } );
 }
 
-
-.job-details dt {
-    margin-top: unit( 10px / @base-font-size-px, em );
+.vacancy-announcement_details dt {
+    margin-top: unit( 12px / @base-font-size-px, em );
     margin-bottom: 0;
 }
+
+.vacancy-announcement_apply {
+    .embed-container {
+        position: relative; 
+        padding-bottom: 56.25%; 
```

Comment:
We have a flexible container mixin I think you could use here https://cfpb.github.io/capital-framework/components/cf-core/#mixins

Reply:
Oh, yeah, that works -- thanks! Updated in https://github.com/cfpb/cfgov-refresh/pull/3784/commits/91043aa558504728cd662a5faca32bea76e71074